docs/src/pages/components/text-fields/ColorTextFields.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { createStyles, makeStyles, Theme } from '@material-ui/core/styles';
+import TextField from '@material-ui/core/TextField';
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    container: {
+      display: 'flex',
+      flexWrap: 'wrap',
+    },
+    textField: {
+      marginLeft: theme.spacing(1),
+      marginRight: theme.spacing(1),
+      width: 200,
+    },
+  }),
+);
+
+export default function ColorTextField() {
+  const classes = useStyles();
+
+  return (
+    <form className={classes.container} noValidate autoComplete="off">
+      <div>
+        <TextField
+          id="standard-basic"
+          className={classes.textField}
```

Comment:
I believe that we could remove the className, only have one on the root form, and remove the intermediary divs. Maybe something like this?
```js
const useStyles = makeStyles(theme => ({
  root: {
    display: 'flex',
    '& > *': {
      width: 200,
      marginTop: theme.spacing(2),
    },
    '& > * + *': {
      marginLeft: theme.spacing(2),
    },
  },
}));
```

Reply:
Should this also be updated for the `BasicTextFields` demo?

Reply:
Hum, maybe we shouldn't change anything here and handle it at the same time we push #16947?

Reply:
I think this effort should be handled in a separate PR. Could be #16947 but easily could be one before then.